book/peer_review.md

```diff
@@ -92,37 +115,51 @@ Be critical and clear, but not mean. Ask questions and set actions.
 
 ```
 
-Internal review should be carried out within your teams. Reviewing code from those with more and less experience is beneficial.
+Internal review should be carried out regularly within the development team. Reviewing code written by those with more and less experience than you is beneficial for both reviewer and developer. Similar questions can be asked from both perspectives, for the reviewer to get a good understanding of the approach and decisions behind the analysis.
+
+## Give timely feedback
 
-## How do we approach code review?
+It's important that feedback is provided in good time, so that the review process does not hold up development of the code and that issues can be addressed before more code is written using the same practices.
```

Comment:
Do we want to really drive it home and explicitly advise against waiting until the end of development for review?

Reply:
Ignore me, you've done this two sections down